bigtable/client/filters_test.cc

```diff
@@ -0,0 +1,76 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "bigtable/client/filters.h"
+
+#include <gmock/gmock.h>
+
+#include "bigtable/client/chrono_literals.h"
+
+/// @test Verify that functions to create simple filters work as expected.
+TEST(FiltersTest, Simple) {
+  namespace btproto = ::google::bigtable::v2;
+
+  auto proto = bigtable::Filter::Latest(3).as_proto();
+  EXPECT_EQ(3, proto.cells_per_column_limit_filter());
+
+  proto = bigtable::Filter::Column("col[A-E]").as_proto();
```

Comment:
Feels like every time you assign a new value to `proto`, it should be a new test, because it entirely resets state.

Reply:
Done.  That is a lot more code, but a valid point.